pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -115,9 +119,7 @@ public SegmentPreprocessingJob(final Properties properties) {
       _pushLocations =
           PushLocation.getPushLocations(StringUtils.split(pushHostsString, ','), Integer.parseInt(pushPortString));
     } else {
-      throw new RuntimeException(String
-          .format("Push location is mis-configured! %s: %s, %s: %s", JobConfigConstants.PUSH_TO_HOSTS, pushHostsString,
-              JobConfigConstants.PUSH_TO_PORT, pushPortString));
+      _pushLocations = null;
```

Comment:
Remove the code of setting _pushLocations and _rawTableName which are already in the BaseSegmentJob. Same for SegmentCreationJob.